elementary/utils/log.py

```diff
@@ -51,3 +61,4 @@ def set_root_logger_handlers(logger_name, files_target_path):
     logger = logging.getLogger(logger_name)
     logger.addHandler(get_file_handler(files_target_path))
     logger.addHandler(get_console_handler())
+    logger.addHandler(log_rotation_handler(files_target_path))
```

Comment:
Shouldn't we replace the `logging.FileHandler` with `logging.RotatingFileHandler` in `get_file_handler` instead?
Now it seems that there are two file handlers and both are writing to the same file.